grass-gis-addons/m.analyse.trees/v.trees.param.worker/v.trees.param.worker.py

```diff
@@ -224,7 +224,7 @@ def crowndiameter(list_attr, treecrowns):
         "v.db.update",
         map=treecrowns,
         column=col_diameter,
-        query_column=f"{col_diameter}/{math.pi}",
+        query_column=f"2*(sqrt({col_diameter}/{math.pi}))",
```

Comment:
```suggestion
        query_column=f"2*(math.sqrt({col_diameter}/{math.pi}))",
```
?

Reply:
I tried this first, but it fails:

`FEHLER: Fehler beim Ausführen von: 'UPDATE trees_temp_0_11005 SET diameter=2*(math.sqrt(diameter/3.141592653589793))'.`

Reply:
Ah right, `sqrt` is here a SQL function, not a python function.